File: 2518/Escada do DINF.py
import math

while True:

    try:
        n = int(input())
        h, c, l = map(int, input().split())

        hipo = math.sqrt(c*c + h*h)
        sizeTotal = n*hipo

        print("%.4f"%((l*sizeTotal)/10000))
    
    except EOFError:
        break

File: 2518/Escada do DINF.cpp
#include <bits/stdc++.h>
#include <math.h>
#include <iostream>

using namespace std;

int main (){

    int n ; 
    
    while (cin >> n){
        int h, c, l; cin >> h >> c >> l;


        double hipo = sqrt(c*c + h*h);
        double sizeTotal = n*hipo;

        printf("%.4f\n", (l*sizeTotal)/10000);
        

    }
}
